common/src/main/kotlin/net/ccbluex/liquidbounce/register/IncludeCommand.kt

```diff
@@ -0,0 +1,29 @@
+/*
+ * This file is part of LiquidBounce (https://github.com/CCBlueX/LiquidBounce)
+ *
+ * Copyright (c) 2015 - 2024 CCBlueX
+ *
+ * LiquidBounce is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * LiquidBounce is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with LiquidBounce. If not, see <https://www.gnu.org/licenses/>.
+ */
+package net.ccbluex.liquidbounce.register
+
+/**
+ * Registers a command as an inbuilt command.
+ * All annotated objects must have a `createCommand()` method that returns the command.
+ *
+ * Only in development versions when [dev] is true.
+ */
```

Comment:
/> All annotated objects must have a `createCommand()` method that returns the command.

I think this is an anti pattern. The user does not know that the `createCommand` method cannot be renamed, Just create an abstract class (like `CommandFactory`) with that method which each command implements. Then complain if a class annotated with `@IncludeCommand` does not implement `CommandFactory`